docs/pages/blog.tsx

```diff
@@ -210,12 +218,7 @@ export default function Blog(props: InferGetStaticPropsType<typeof getStaticProp
           >
             {[firstPost, secondPost].map((post) => (
               <Paper
-                component={Link}
```

Comment:
@siriwatknp I was thinking of having the whole paper component here, in the case of the featured posts, be a link! Do you think it's feasible?

Reply:
Yes, will fix it.

Reply:
I will use this reference https://inclusive-components.design/cards/ for accessibility of the card.

Reply:
Oh nice, thanks for sharing! Didn't know this one :)

Reply:
@danilo-leal I have updated the code, please take a look.

Reply:
I'm not being able to run the branch locally and I don't know how to proceed 😕 
When using `yarn docs:dev`, even after a `yarn install` an error appears saying that it's not possible to find the `'./src/featureToggle'` module.

Reply:
next.config.js cannot read `.ts`. I have changed the feature toggle to `.js`. It should work now.